pages/infrastructure_providers-validators/running_full_node.md

```diff
@@ -1,73 +1,60 @@
-
-https://services.lavenderfive.com/mainnet/dydx#live-peers
-
-https://polkachu.com/live_peers/dydx
-
-Update `persistent_peers`  in the config.toml file to include a randomly selected list of 5 peers from the list of live peers obtained using the links above
-
-### Snapshots
-
-Snapshots contain a compressed copy of the chain data which allow the full node to bootstrap to a recent state in the blockchain. A list of snapshot services can be found [here](../infrastructure_providers-network/resources.mdx#snapshot-service).
-
-### State sync (Alternative to snapshots)
-
-State Sync enables a new node to join the network by obtaining a snapshot of the application state from a state sync node at a recent height. This eliminates the need to fetch and replay all historical blocks. A list of state sync services with instructions are listed below
-
-https://polkachu.com/state_sync/dydx
-
-https://services.lavenderfive.com/mainnet/dydx/statesync
-
-https://autostake.com/networks/dydx/#state-sync
-
-### Address Book
+## Save an Address Book
+Download the latest `addrbook.json` file, which stores configuration details that help your node efficiently connect to peers in its network. Download an up-to-date address book file for your deployment from an [Address Book](../infrastructure_providers-network/resources#address-book-providers) provider.
 
-The **`addrbook.json`** file is used to store configuration details that help a node connect to other peers in the network more efficiently. This can be obtained from the one of the below services and needs to be stored in the `config` folder.
+Save the `addrbook.json` file in your `/.dydxprotocol/config` directory.
 
-https://polkachu.com/addrbooks/dydx
+## Back Up Your Node
+> If you followed the procedure on the previous page, [Set Up a Full Node](../infrastructure_providers-validators/how_to_set_up_full_node.md), you already have a snapshot installed.
 
-https://services.lavenderfive.com/mainnet/dydx#latest-addrbook
+Your full node needs a backup plan to replay the history of the network in case it falls out of sync. You can back up your node in one of two ways.
 
-https://autostake.com/networks/dydx/
+### Snapshot
+You can use a **snapshot** stored on the system that your node runs on. A snapshot contains a compressed copy of the application state at the time the snapshot was taken. If your node falls out of sync, a snapshot allows the node to recover to that saved state before replaying the rest of the history of the network. This speeds up the syncing process because you avoid replaying the entire history of the network, instead starting from your stored application state snapshot. To use a snapshot to back up your full node, install a snapshot for your deployment from a [Snapshot Service](../infrastructure_providers-network/resources.mdx#snapshot-service).
```

Comment:
I think this is a bit misleading. It currently reads like "to back up your node, you can use a snapshot service." However, this is about "how can you use snapshot to restore your full node to the healthy + latest state"

Can you update it accordingly?